docs/01-engage/02-accounts/02-creating-accounts/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Creating Accounts
+sidebar_position: 2
+---
+
+Accounts in **ENGAGE** are separate individuals (donors or supporters) and organizations that donate charity. This article will walk you through creating a new account via ENGAGE.
+
+## Option 1:1 Create an Account from the Dashboard using Create
+
+- On the [ENGAGE dashboard](https://beta.n3o.cloud/eu1/engage/dashboard), click the blue button **Create**. A bar containing different icons will be shown.
+
+![Create Account Dashboard](./create-account-dashboard.png)
+
+- Click the **Create Account** button. This will take you to the create account screen.
+
+## Option 1:2 Create an Account from the Dashboard using Quick Navigation
+
+- On the [ENGAGE dashboard](https://beta.n3o.cloud/eu1/engage/dashboard), navigate to **Quick Navigation**.
+
+![Create Account Quick Navigation](./create-account-quick-navigation.png)
+
+- Click on **Create Account** and a screen will appear to create the account.
+
+:::note 
+An account can be created for both individual and an organization. Each of them will have different fields to be covered. To follow along, make sure to have the correct information. 
+:::
+
+## Create an Individual Account
+
+:::note Important 
+- Input boxes with *asterisk* are compulsory to fill in. If you do not, an error message will appear e.g. *"State is a required field"*. 
+- By default, create new account screen always opens for an individual account.
+- Sometimes, while entering account information, background search might show previously entered accounts with a similar input.   
+:::
+
+1. In the *Name* section, specify the **Title**, **First Name** and **Last Name**.
+
+![Create Account Name](./create-account-1.png)
+
+2. Next, in the *Address* section, enter the **Country** you reside in, and it's respective **Postcode**. Input your address as **Line 1**, **Line 2** or **Line 3** and the **Town/City**.
+
+:::info
+- Most of the time, the address may appear relative to the postcode search.
+- For some countries, address input parameters might be different.
+:::
+
+![Create Account Address](./create-account-2.png)
+
+3. Moving forward, in the *Phone Number* section, enter your phone number along with the country code. Then select whether the number is a *fixed line* or *mobile*. You can also add extra phone numbers with **+ Add Phone Number** and remove them with the **Remove** button. 
```

Comment:
Again, not your phone number but accounts number.

Should tell them the format to enter it in, e.g. do I put +447777123 or 07777123. You can also say that most of the time the system will be able to detect if the number is a fixed line or mobile, but if not, you should set this manually if known.

Need to also explain the first number will become the primary which is the one used by automated processes in the system.